x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_count_panel/field_count_panel.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiNotificationBadge, EuiSwitch, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { FC } from 'react';
+
+interface Props {
+  metricsStats?: {
+    visibleMetricFields: number;
+    totalMetricFields: number;
+  };
+  fieldsCountStats?: {
+    visibleFieldsCount: number;
+    totalFieldsCount: number;
+  };
+  showEmptyFields: boolean;
+  toggleShowEmptyFields: () => void;
+}
+export const FieldCountPanel: FC<Props> = ({
+  metricsStats,
+  fieldsCountStats,
+  showEmptyFields,
+  toggleShowEmptyFields,
+}) => {
+  return (
+    <EuiFlexGroup
+      alignItems="center"
+      gutterSize="xs"
+      style={{ marginLeft: 5 }}
```

Comment:
not sure if we ever used this value for margins

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/4ea6d174728033f29d81392da47db1f3b409d4a5